pkg/manager/impl/execution_manager.go

```diff
@@ -55,19 +55,22 @@ const childContainerQueueKey = "child_queue"
 type projectDomainScopedStopWatchMap = map[string]map[string]*promutils.StopWatch
 
 type executionSystemMetrics struct {
-	Scope                    promutils.Scope
-	ActiveExecutions         prometheus.Gauge
-	ExecutionsCreated        prometheus.Counter
-	ExecutionsTerminated     prometheus.Counter
-	ExecutionEventsCreated   prometheus.Counter
-	PropellerFailures        prometheus.Counter
-	PublishNotificationError prometheus.Counter
-	TransformerError         prometheus.Counter
-	UnexpectedDataError      prometheus.Counter
-	SpecSizeBytes            prometheus.Summary
-	ClosureSizeBytes         prometheus.Summary
-	AcceptanceDelay          prometheus.Summary
-	PublishEventError        prometheus.Counter
+	Scope                     promutils.Scope
+	ActiveExecutions          prometheus.Gauge
+	ExecutionsCreated         prometheus.Counter
+	ExecutionsTerminated      prometheus.Counter
+	ExecutionEventsCreated    prometheus.Counter
+	PropellerFailures         prometheus.Counter
+	PublishNotificationError  prometheus.Counter
+	TransformerError          prometheus.Counter
+	UnexpectedDataError       prometheus.Counter
+	SpecSizeBytes             prometheus.Summary
+	ClosureSizeBytes          prometheus.Summary
+	AcceptanceDelay           prometheus.Summary
+	PublishEventError         prometheus.Counter
+	WorkflowBuildSuccess      prometheus.Counter
+	WorkflowBuildFailure      prometheus.Counter
+	TerminateExecutionFailure prometheus.Counter
```

Comment:
Since these are counters, should we use plurals similar to PropellerFailures

Reply:
okay, done